www/js/blockly-api.js

```diff
@@ -1,31 +1,32 @@
 /*----- DEFINE API -----*/
 
-leftMotor = {FORWARD: "XIO-P0", BACKWARD: "XIO-P1"};
-rightMotor = {FORWARD: "XIO-P6", BACKWARD: "XIO-P7"};
-motorPins = {LEFT: leftMotor, RIGHT: rightMotor};
+leftMotor = { FORWARD: "XIO-P0", BACKWARD: "XIO-P1" };
+rightMotor = { FORWARD: "XIO-P6", BACKWARD: "XIO-P7" };
+motorPins = { LEFT: leftMotor, RIGHT: rightMotor };
 
 function initApi(interpreter, scope) {
 
 	// Add an API function for the alert() block.
-	var wrapper = function(text) {
+	var wrapper = function (text) {

```

Comment:
this was the auto-formatting from vscode. I personally like it though, I'm all about giving symbols room to breathe.